sklearn/ensemble/bagging.py

```diff
@@ -541,6 +541,39 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
         `oob_decision_function_` might contain NaN. This attribute exists
         only when ``oob_score`` is True.
 
+
+    Examples
+    --------
+    >>> from sklearn.dummy import DummyClassifier
+    >>> from sklearn.ensemble import BaggingClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_samples=1000, n_features=4,
+    ...                            n_informative=2, n_redundant=0,
+    ...                            random_state=0, shuffle=False)
+    >>> clf = BaggingClassifier(n_estimators=100, random_state=0)
+    >>> clf.fit(X, y)  # doctest: +NORMALIZE_WHITESPACE
+    BaggingClassifier(base_estimator=None, bootstrap=True,
```

Comment:
It's odd that some fail and some don't, but for the example, I'd just put the fit right after construction of the class and put it in `clf`, and as a result you don't get the pprint of the object, and that should solve the issue anyway.